spec/factories/utils.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  trait :discarded do
```

Comment:
what about renaming this to `deleted`?  😬

Reply:
well, sure :)